en/publish/publish-qqplay.md

```diff
@@ -1,67 +1,66 @@
-- 支持 QQ「玩一玩」平台标准的远程资源加载、缓存，以及缓存资源的版本控制
+{% raw %}
 
-## 使用 Cocos Creator 一键发布手机 QQ「玩一玩」
+<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=m055288q7cl&tiny=0&auto=0" allowfullscreen></iframe >
 
-**准备工作：**
+{% endraw %}
 
-- 获取 Bricks 引擎工程（这个工程暂时不能对外公开，开发者可以向 Creator 引擎团队申请内测，或者直接向腾讯申请 bitbucket 库读取权限）
+<br>
 
-**发布流程：**
+__QQ Play__ is similar to WeChat games, but built for __QQ__. __QQ Play__ allows you to play a variety of popular mobile games *without downloading* them. The underlying runtime of playing is not a real HTML browser kernel, but a self-developed Bricks game engine. __Cocos Creator__ has supported game content cross-platform publishing to __QQ Play__ since v1.9.
 
-1、使用 Creator 打开需要发布的项目工程，在构建发布面板中选择 [QQ Play] 平台。`远程服务器地址` 选项，若有需要可自行设置。然后点击 **构建**。
+ As a cross-platform game engine, __Cocos Creator's__ main tasks for users include:
 
-![](./publish-qqplay/build.jpeg)
+- The engine is responsible for adapting the __QQ Play__ platform, the user's game logic code does not require any additional modifications.
+- The __Cocos Creator__ editor provides a packaging process that can be directly published as a game that meets the __QQ Play__ specification.
+- Support remote __QQ__ resource loading, caching, and version control of cache resources for __QQ Play__ platform standard.
 
-构建完成后访问项目文件夹，在 build 目录下就会生成一个 qqplay 的发布包：
+## Use __Cocos Creator__ to launch a mobile phone with one click __QQ Play__
 
-![](./publish-qqplay/package.jpeg)
+**Ready to work:**
 
-2、将 qqplay 文件夹中的内容拷贝到 Bricks 引擎工程的 Res 文件夹中：
+Obtain the Bricks engine project (this project is temporarily unavailable, developers can apply to the __Cocos Creator__ engine team for internal testing, or directly apply to Tencent for bitbucket library read access)
 
-![](./publish-qqplay/paste.jpeg)
+**Publishing process:**
 
-3、打开 Bricks 引擎中的 Xcode 工程，就可以直接进行玩一玩的编译调试了。具体可查看 [Bricks 引擎官方调试文档](http://hudong.qq.com/docs/engine/introduce/safari_debug.html)。
+1. Use __Cocos Creator__ to open the project project you want to publish, and select __QQ Play__ platform in the __Build...__ panel. If necessary, you can set the `Remote server address` option. Then click on **Build**.
 
-与这篇文档搭配的还有一段教学视频，可以结合视频中的操作来理解，更加直观。
+![](./publish-qqplay/build.jpeg)
 
-{% raw %}
+After the build is complete, a __qqplay__ release package will be generated in the __build__ directory:
 
-<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=m055288q7cl&tiny=0&auto=0" allowfullscreen></iframe>
+![](./publish-qqplay/package.jpeg)
 
-{% endraw %}
+2. Copy the contents of the __qqplay__ folder to the __Res__ folder of the Bricks Engine project:
 
-<br>
+![](./publish-qqplay/paste.jpeg)
+
+3. Open the Xcode project in the Bricks engine, you can directly play and compile and debug. For details, see the [Bricks Engine Official Debugging Document](http://hudong.qq.com/docs/engine/introduce/safari_debug.html).
```

Comment:
"you can directly play and compile and debug." -> "you can compile and debug the QQ Play directly"